velox/functions/prestosql/tests/TimestampWithTimeZoneCastTest.cpp

```diff
@@ -185,10 +194,16 @@ TEST_F(TimestampWithTimeZoneCastTest, fromVarcharInvalidInput) {
       "Unknown timezone value: \"America/California\"");
   VELOX_ASSERT_THROW(
       testCast(invalidStringVector3, expected),
-      "Unable to parse timestamp value: \"2012-10-31foo01:00:47 America/Los_Angeles\"");
+      "Unknown timezone value: \"foo01:00:47\"");
   VELOX_ASSERT_THROW(
       testCast(invalidStringVector4, expected),
-      "Unable to parse timestamp value: \"2012-10-31 35:00:47 America/Los_Angeles\"");
+      "Unknown timezone value: \"35:00:47\"");
+  VELOX_ASSERT_THROW(
+      testCast(invalidStringVector5, expected),
+      "Unable to parse timestamp value: \"2012 America/Los_Angeles\"");
+  VELOX_ASSERT_THROW(
+      testCast(invalidStringVector6, expected),
+      "Unable to parse timestamp value: \"2012-10 America/Los_Angeles\"");
```

Comment:
this is year,month+TZ, is this invalid because its not kIso8601 mode?

Reply:
That's right, for casting we use the "Presto" parse mode.